server/test/controllers/role.specs.js

```diff
@@ -188,44 +222,58 @@ describe('Role Endpoints', () => {
     });
   });
 
-  describe('delete role function', () => {
-    it('should delete role if is a super admin', (done) => {
+  describe('Delete role function', () => {
+    it(`should return a success message when a user makes a request
```

Comment:
Please review the language here. I believe this test spec is to communicate what happens when a Super Admin tries to delete a role.